### PR TITLE
[Serve] RayServe TF, PyTorch, Sklearn Examples

### PR DESCRIPTION
**Pull request by simon-mo**
## Why are these changes needed?

This is built on the PyTorch example PR. Please only review the relevant sklearn part.

## Checks

- [ ] I've run `scripts/format.sh` to lint the changes in this PR.
- [ ] I've included any doc changes needed for https://docs.ray.io/en/latest/.
- [ ] I've made sure the tests are passing. Note that there might be a few flaky tests, see the recent failure rates at https://ray-travis-tracker.herokuapp.com/.
- Testing Strategy
   - [ ] Unit tests
   - [ ] Release tests
   - [ ] This PR is not tested (please justify below)

### CODE REVIEW

File: doc/source/index.rst [hunk 1]
```diff
@@ -287,6 +287,9 @@ Getting Involved
    :caption: RayServe
 
    rayserve/overview.rst
+   rayserve/tensorflow-tutorial.rst
```
Comment (anabranch) [inline doc/source/index.rst:290]: please put in `/tutorials/tensorflow.rst`. Please apply to the others as well.

File: python/ray/serve/BUILD [hunk 2]
```diff
@@ -50,3 +50,27 @@ py_test(
...
+py_test(
+    name = "tutorial_tensorflow",
+    size = "small",
+    srcs = glob(["examples/doc/*.py"]),
+    tags = ["exclusive"],
+    deps = [":serve_lib"]
+)
+
+py_test(
+    name = "tutorial_pytorch",
```
Comment (anabranch) [inline python/ray/serve/BUILD:63]: Will this name potentially cause conflicts or are they namespaced to `serve`?

**Review by bllchmbrs — APPROVED** [review]
two nits then LGTM

File: python/ray/serve/BUILD (hunk 2 above)
Comment (simon-mo) [reply python/ray/serve/BUILD:63]: These are namespaced

**Pull request merged**